backend/src/routes/index.ts

```diff
@@ -8,7 +8,7 @@ const router = express.Router();
 routes.forEach((route) => {
   import(route.handler)
     .then((module) => {
-      const HandlerClass = module.default;
+      const HandlerClass = module[route.handlerClass];
```

Comment:
Yeah, I was getting the error

`[1] Failed to load route '/actions/tables.join': TypeError: HandlerClass is not a constructor
[1]     at /Users/TMF/Desktop/Code/personal_projects/poker-react-node/backend/src/routes/index.ts:12:31
[1]     at processTicksAndRejections (node:internal/process/task_queues:95:5)
`
I think it was working before because the actions file had a default class that was exported? And since there are now multiple we had to remove the default and just export all the classes individually. At least that's my guess.

Reply:
ah got it, I can't see that you added a new one in this PR? but anyway when you do: 

What we want to do is refactor this by:
1. Rename `actions.ts` to the handler class name
2. Do the default export at the bottom for each new one (as per the current one in `actions.ts`
3. Create new handlers in this folder and then ref them in routeConfig:

```
...
 handler: '../handlers/actions/TablesJoinHandler.ts
...
```

I think that should work?

Reply:
Ok yeah sweet we can do different files instead of putting them all in actions.ts file.